### ISSUE TITLE
Get this version of deposit-cli linked in launchpad app

### ISSUE THREAD

**Issue by maraoz** [ethstaker/ethstaker-deposit-cli#134]
deposit-cli repo by EF is [unmaintained](https://github.com/ethereum/staking-deposit-cli/pull/397).
We should get the [launchpad app](https://launchpad.ethereum.org/en/) to link to this version.

**Comment by yorickdowne** [comment]
Yes and: We need to get through audits. We have a few medium-large changes to this tool in the issue list yet. Audit kicks off in October.

After audit and when all issues raised by audit have been addressed - by code or a comment - we can then release a “generally available” version and talk to the launchpad maintainers about their appetite for listing this fork.

**Comment by yorickdowne** [comment]
We are now linked from launchpad

https://launchpad.ethereum.org/en/generate-keys
